.travis.yml

```diff
@@ -1,7 +1,7 @@
 dist: trusty
 sudo: required
 # xcode8 has jdk8
-osx_image: xcode8
+osx_image: xcode10.1
```

Comment:
Is this necessary? If so, can you remove the comment above as it's no longer applicable?